index.php

```diff
@@ -1,53 +1,67 @@
-<?php require_once __DIR__ .'/variables.php' ?>
+<?php
+require_once __DIR__ . '/variables.php';
+?>
 <!DOCTYPE html>
 <html lang="en">
-<?php require_once __DIR__ .'/template/head.phtml';?>
-<title><?=$titleIndex?></title>
+<?php
+require_once __DIR__ . '/template/head.phtml';
+?>
+<?php

```

Comment:
In the case where you have 2 PHP require parts do not need to close `?>` and open a new tag `<?php` . You can combine code into one tag

Reply:
done